README.md

```diff
@@ -1,25 +1,53 @@
-This is similar to React's `Create React App`. The idea is to provides a
+
+# Create Socket App 
+
+`Create Socket App` is similar to React's `Create React App`. The idea is to provides a
 few basic boilerplates and some strong opinions so you can get coding on a
 production quality app as quickly as possible.
 
-## Usage: 
+## Getting Started 
+
+Before getting started, you may need some `Prerequisites`, to learn more you can check
+[Socket Runtime](https://sockets.sh/).
+**You'll also need to have Node>= 16 on your local developement machine.**
+That's it, you are ready to go!
+
+To create a new app, you may choose one of the following methods: 
+
+With **npm**:
 
-With npm:
+Any of these following commands will work
 ```bash
+npx create socket-app [react | svelte | tonic | vanilla | vue]
+npm init socket-app [react | svelte | tonic | vanilla | vue]
 npm create socket-app [react | svelte | tonic | vanilla | vue]
```

Comment:
I think that we need to choose only one between `npm init socket-app` or `npm create socket-app`